### DISCUSSION TITLE
[connector-request] Youtube Data API

### DISCUSSION THREAD

**Discussion by bala-ceg** [airbytehq/airbyte#46738] [New Connector Request]
This is a suggestion to create a connector for Youtube Data API using the UI Connector Builder.
The website for the connector is  [Youtube Data API](https://developers.google.com/youtube/v3/guides/implementation). 

If you want to try, claim this issue and start working on it following the steps below.

**Steps:**

- Comment in the issue and wait to be assigned to start working on it.
- Map the API endpoints (get approval to move to 3 step)
- Describe steps to get credentials.
- Create the connector using the UI Builder

**Comment by bala-ceg** [comment]
@marcosmarxm Airbyte currently supports [youtube-analytics](https://docs.airbyte.com/integrations/sources/youtube-analytics), but it does not support [Youtube Data API](https://developers.google.com/youtube/v3/guides/implementation)

I am planning to implement the below endpoints 

[Retrieve a list of channel activities](https://developers.google.com/youtube/v3/guides/implementation/activities#activities-list-channels)
[Retrieve a list of subscription activities](https://developers.google.com/youtube/v3/guides/implementation/activities#activities-list-subscriptions)
[Retrieve a list of caption tracks for a video](https://developers.google.com/youtube/v3/guides/implementation/captions#captions-list)
[Retrieve information about a channel](https://developers.google.com/youtube/v3/guides/implementation/channels#channels-list)
[Retrieve a channel's uploaded videos](https://developers.google.com/youtube/v3/guides/implementation/videos#videos-retrieve-uploads)
[Retrieve a channel's subscriptions](https://developers.google.com/youtube/v3/guides/implementation/subscriptions#subscriptions-retrieve-for-channel)

**Comment by bala-ceg** [comment]
@marcosmarxm can you please assign this issue to me, i will work on it as a part of the hackathon

**Reply by marcosmarxm** [reply]
Please create an issue.

[a comment by an automated account is not shown]
